### PR TITLE
Configure custom repo for Debian buster

### PR DESCRIPTION
**Pull request by McJoppy**

### CODE REVIEW

File: manifests/repo.pp [hunk 1]
```diff
@@ -8,18 +8,27 @@
 
   ensure_packages(['apt-transport-https', 'lsb-release', 'ca-certificates'], {'ensure' => 'present'})
 
-  if ($::operatingsystem == 'Ubuntu') {
+  if ($os::name == 'Ubuntu') {
```
Comment (brettt89) [inline manifests/repo.pp:11]: Why were these changes needed? Have they been tested on AWS as well as CCL?

Comment (McJoppy) [reply manifests/repo.pp:11]: Nothing tested yet - but `operatingsystem` is a puppet legacy fact.

Need to test in both platforms still

Comment (brettt89) [reply manifests/repo.pp:11]: It might be better to put this change into a different change so you are not blocking the PHP fix with these changes needing to be tested

**Review by brettt89 — APPROVED** [review]
